Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst

```diff
@@ -147,8 +147,9 @@ to this wizard (via page TSconfig).
 
    Starting with TYPO3 version 12.0 file :file:`EXT:examples/Configuration/page.tsconfig`
    is automatically included. For version 11.5 and below this file has to be included in the
-   :file:`ext_localconf.php`. See :ref:`Setting global page TSconfig, compatible with TYPO3
-   11 and 12 <t3tsconfig:page-tsconfig-v11-v12>`.
+   :file:`ext_localconf.php`.  
+   See :ref:`Setting global page TSconfig <t3tsconfig:pagesettingdefaultpagetsconfig>`.
+   See :ref:`Wizard configuration reference <t3tsconfig:pagenewcontentelementwizard>`-
```

Comment:
Is the dash ('-') here at the end intentional?

Reply:
No, it was a typo, thanks for the hint.
It's corrected in the PR.